docs/Agent/user-guide/man_page.md

```diff
@@ -315,17 +324,29 @@ Show this message and exit.
 `pbench-register-tool-set` - register the specified toolset
 
 **SYNOPSIS**  
-`pbench-register-tool-set [--group=<group-name>] [--interval=<interval>] [--no-install] [--remotes=<remote-host>[,<remote-host>]] [--labels=<label>[,<label>]] [--help]`
-
-`pbench-register-tool-set [--group=<group-name>] [--interval=<interval>] [--no-install] [--remotes=@<remotes-file>] [--help]`
+`pbench-register-tool-set [OPTIONS]`
 
 **DESCRIPTION**  
 Register all the tools in the specified toolset.
 
+Available <`tool-set`> from /opt/pbench-agent/config/pbench-agent.cfg:
+
+- heavy
+- legacy
+- light
+- medium
+
 **OPTIONS**
+
+`<tool-set>`
+A named tool set argument from one of: `heavy`, `legacy`, `light`, `medium`.
```

Comment:
We probably don't need lines 341-342, given that we have lines 332-337.  (Removing lines 341-342 makes sense, since `<tool-set>` is actually a required command line _argument_ and not an _option_.)